proj-xcode/PowerAuth2/PowerAuthLog.m

```diff
@@ -82,6 +85,22 @@ BOOL PowerAuthLogIsVerbose(void)
 #endif
 }
 
+void PowerAuthLogToConsole(BOOL enabled)
+{
+#ifdef ENABLE_PA2_LOG
+    s_log_to_console = enabled;
+#endif
+}
+
+BOOL PowerAuthLogToConsoleEnabled(void)
```

Comment:
I would keep consistent naming with other functions in the file:

- `PowerAuthLogToConsoleSetEnabled(BOOL)`
- `PowerAuthLogToConsoleIsEnabled(void)`

Reply:
Yop 👍